src/prototype_room_creepbuilder.js

```diff
@@ -1,32 +1,20 @@
 'use strict';
 
-Room.prototype.spawnCreateCreep = function(role, heal, level, squad, routing) {
-  var energy = this.energyAvailable;
-
+Room.prototype.spawnCreateCreep = function(creep) {
```

Comment:
https://github.com/TooAngel/screeps/blob/master/src/config_brain_squadmanager.js#L98 as an example. Bonus: If you find something automatic for atom, or even a check as grunt task.